synfig-studio/src/gui/workarea.cpp

```diff
@@ -2179,6 +2179,16 @@ WorkArea::set_selected_value_node(etl::loose_handle<synfig::ValueNode> x)
 	}
 }
 
+void
+WorkArea::set_active_bone_value_node(etl::loose_handle<synfig::ValueNode> x)
```

Comment:
Wouldn't be good to check if the valuenode refers to a Bone?

Reply:
Yes! I'm working on modifying them. Currently, I'm stuck with Undo system and adding deformation layer 😅 I'll get onto this asap

Reply:
This issue is still present in current code.

Reply:
Resolved in latest commit.